.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+  - 2.2.2
```

Comment:
Can you open up the range of rubies we test on?  Say 2.0+

Also if you add `sudo: false` it should build faster.

@jmanero-r7 care to weigh in?